app/components/publications/publication/publication-activities/publication-edit-modal.js

```diff
@@ -49,14 +41,4 @@ export default class PublicationEditModal extends Component {
       publicationDate: this.publicationDate,
     });
   }
-
-  @dropTask
```

Comment:
@erikap Deze `@dropTask` leek me (hier en binnen `publication-registration-modal.js`) in zijn geheel niet meer nodig? Mocht dat toch niet het geval zijn, laat zeker iets weten!

Reply:
Klopt helemaal :+1: